### PR TITLE
 improve mobile layout for Modal component

### PR DESCRIPTION
**Pull request by alsturm**
mobile UX enhancement.

### CODE REVIEW

**Comment by taulinger** [comment]
- ported the Modal component to MvuElement
- added tests for orientation regards

Would suggest inviting @thiloSchlemmer for the final review

File: test/modules/modal/components/Modal.test.js [hunk 1]
```diff
@@ -30,49 +57,105 @@ describe('Modal', () => {
...
-			//Note: Webkit appends a line break to the 'content' in this case
-			expect(element.shadowRoot.querySelector('.modal__content').innerText).toMatch(/content[\r\n]?/);
+				openModal('title', 'content');
+
+				expect(store.getState().modal.data.title).toBe('title');
+				expect(store.getState().modal.data.content).toBe('content');
+				expect(element.shadowRoot.querySelector('.modal')).toBeTruthy();
+				expect(element.shadowRoot.querySelector('.modal__title').innerText).toMatch(/title[\r\n]?/);
+				//Note: Webkit appends a line break to the 'content' in this case
+				expect(element.shadowRoot.querySelector('.modal__content').innerText).toMatch(/content[\r\n]?/);
```
Comment (thiloSchlemmer) [inline test/modules/modal/components/Modal.test.js:106]: a updated 'Note'-comment  would be nice...i suggest to mention both expectations or use a more generalized form, to stay in touch with the changes

File: test/modules/modal/components/Modal.test.js [hunk 2]
```diff
@@ -30,49 +57,105 @@ describe('Modal', () => {
...
+
+				const template = (str) => html`${str}`;
+
+				openModal('title', template('content'));
+
+				expect(store.getState().modal.data.title).toBe('title');
+				expect(store.getState().modal.data.content).toBeInstanceOf(TemplateResult);
+				expect(element.shadowRoot.querySelector('.modal')).toBeTruthy();
+				expect(element.shadowRoot.querySelector('.modal__title').innerText).toMatch(/title[\r\n]?/);
+				//Note: Webkit appends a line break to the 'content' in this case
```
Comment (thiloSchlemmer) [inline test/modules/modal/components/Modal.test.js:126]: same here, see comment above

**Review by thiloSchlemmer — APPROVED** [review]
nice improvement! 

...negligible comments inline

**Pull request merged**